src/modules/systemlib/battery.cpp

```diff
@@ -166,33 +167,19 @@ Battery::sumDischarged(hrt_abstime timestamp, float current_a)
 void
 Battery::estimateRemaining(float voltage_v, float current_a, float throttle_normalized, bool armed)
 {
+	// correct battery voltage locally for load drop to avoid estimation fluctuations
 	const float bat_r = _param_r_internal.get();
-
-	// remaining charge estimate based on voltage and internal resistance (drop under load)
-	float bat_v_empty_dynamic = _param_v_empty.get();
-
 	if (bat_r >= 0.0f) {
```

Comment:
Might as well use  _param_r_internal.get() and _param_v_empty.get() directly.

Reply:
`_param_v_empty.get()` already gets used directly now.
`bat_r` made it more readable in my opinion but you're right I should rename the block param to be more readable, that's better.
Thanks for already looking at it 👍